stack-shell.nix

```diff
@@ -9,6 +9,7 @@ let
     zlib
     gmp
     pkgconfig
+    ncurses
   ] ++ lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [
```

Comment:
No it is not. It was just a missing dependency on the Nix script that prevented me to compile Idris. It kinda slipped out.